http/mux.go

```diff
@@ -85,6 +86,7 @@ func NewMuxer() ResolverMuxer {
 		w.WriteHeader(http.StatusNotFound)
 		enc.Encode(NewErrorResponse(ctx, fmt.Errorf("404 page not found"))) // nolint:errcheck
 	}))
+	r.Use(middleware.RedirectSlashes)
```

Comment:
https://github.com/go-chi/chi/blob/1129e362d6cce6e3805e3bc8dfbaeb34b5129789/middleware/strip_test.go#L105-L107

Reply:
Thank you. Fixed. 3fbf066e93645b3640f49a1abd29c4cc8a1299fd